framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -628,6 +628,7 @@ private void parseMethod(
                 // workaround
                 assert paramType.getKind() == TypeKind.ARRAY;
                 annotate(((AnnotatedArrayType) paramType).getComponentType(), param.getType());
```

Comment:
Isn't it strange to use the component type for one, but the top-level type for the other?